### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by JenniClark** [community/community#111418] [Other Feature Feedback, Questions, & Ideas]
hi

**Comment by ettaboyle** [comment]
Hi @JenniClark,

Thanks for being a part of the GitHub Community, we're glad you're here!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions.  

I can see that you've created another discussions [help me #110821](https://github.com/orgs/community/discussions/110821) that aligns with our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), but this post and [Help css #113312](https://github.com/orgs/community/discussions/113312) are both off topic & non-sensical.

**We ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.** 

Best of luck!
